test/package_handler_test.rb

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env rspec
+#
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "spec_helper"
+require "y2storage/package_handler"
+
+describe Y2Storage::PackageHandler do
+  context("without adding any devicegraph's storage features") do
+    subject { described_class.new }
+
+    it "has an empty package list" do
+      expect(subject.pkg_list).to be_empty
+    end
+  end
+
+  context("using a devicegraph with Btrfs and LVM") do
+    let(:dg_features) { ::Storage::UF_BTRFS | ::Storage::UF_LVM }
+    let(:devicegraph) { instance_double("::Storage::devicegraph", used_features: dg_features) }
```

Comment:
Does this work with `instance_double("::Storage::devicegraph")` instead of `instance_double("::Storage::Devicegraph")` (beware capitalization of Devicegraph)? Good to know